content-signup/README.md

```diff
@@ -0,0 +1,73 @@
+# Automatically send an email following up with content to people who signup via a form.
+
+For marketing activities, you may want to automate the sending of key links dynamically to content such as videos, whitepaper links, blogs, PDF summaries, etc based on what people mark as their specific interest in your contact-us form linked on your website, email signature, or wherever you share the form's link.
```

Comment:
The transformation gallery version references using Google Docs, do you want to as well for the content per topic?